src/Tizen.NUI.Components/Controls/DropDown.cs

```diff
@@ -1336,6 +1337,10 @@ protected override void Dispose(DisposeTypes type)
                 base.Dispose(type);
             }
 
+            /// <summary>
+            /// Get DropDown item style.
+            /// </summary>
+            /// <returns>The empty.</returns>
```

Comment:
Minor suggestion.
Could you make comment for GetViewStyle() in each component same?

They are all different..